my-app/src/main/java/com/JavaWebServer/app/RESTMethods/Post.java

```diff
@@ -0,0 +1,17 @@
+package com.JavaWebServer.app;
+
+public class Post implements RestMethod {
+  private final String METHODKEY = "POST";
+  private final String ACCEPTED ="HTTP/1.1 200 ok";
+
+  public Post() { }
+
+  public String handleRequest(Request request) {
+    if ((request.getMethod()).equals(METHODKEY)) {
```

Comment:
Also, if you think about it from the standpoint of keeping your code DRY, every "RequestHandler" is going to need to do this, and I would not recommend adding a ton of `if` `else` statements in every `RestMethod`